core/modules/parsers/wikiparser/rules/list.js

```diff
@@ -153,28 +153,54 @@ exports.parse = function() {
 	return [listStack[0]];
 };
 
-exports.serialize = function(tree, serialize) {
-	// Serialize the list recursively
-	return serializeList(tree, serialize, 0);
-};
+exports.serialize = function (tree, serialize) {
+	// Helper function to find the marker for a given list container tag and item tag
+	function findMarker(listTag, itemTag) {
+		for(var key in listTypes) {
+			if(listTypes[key].listTag === listTag && listTypes[key].itemTag === itemTag) {
+				return key; // Return the marker associated with the list tag and item tag
+			}
+		}
+		return ''; // Return empty string if no matching marker is found
+	}
+
+	// Recursive function to serialize list nodes, handling nested lists and formatting output
+	function serializeList(node, markerPrefix) {
+		var result = [];
+		if(node.type === 'element' && ['ul', 'ol', 'dl', 'blockquote'].includes(node.tag)) {
```

Comment:
TW code uses double quotes for strings `"` instead of `'`

Reply:
This should be added to eslint. I see the space after if is in eslint.

Reply:
PR has been merged already